stacks/services/GeneratorBucket.ts

```diff
@@ -0,0 +1,22 @@
+import { BucketEncryption } from "aws-cdk-lib/aws-s3";
+import { Bucket, Stack } from "sst/constructs";
+
+export const createGeneratorBucket = (stack: Stack): Bucket => {
```

Comment:
We could maybe just have a createBucket function and allow passing in the name and whether versioning is enabled, they should always be encrypted and have blockPublicAccess enabled

Reply:
done